python/dgl/transform.py

```diff
@@ -148,40 +223,49 @@ def segmented_knn_graph(x, k, segs):
     return g
 
 def to_bidirected(g, readonly=None, copy_ndata=False):
-    r""" Convert the graph to a bidirected one.
+    r"""Convert the graph to a bidirectional simple graph, adding reverse edges and
+    removing parallel edges.
 
-    The function generates a new graph with no edge features.
-    If g has an edge for i->j but no edge for j->i, then the
-    returned graph will have both i->j and j->i.
+    The function generates a new graph with no edge features.  In the new graph,
+    a single edge ``(u, v)`` exists if and only if either an edge connecting ``u``
+    to ``v`` or an edge connecting ``v`` to ``u`` exists in the original graph.
```

Comment:
"if either an edge connecting ``u`` to ``v`` or an edge connecting ``v`` to ``u`` exists" -> "if there is an edge connecting ``u`` to ``v`` or an edge connecting ``v`` to ``u``"